templates/storyeditview/passageEditDSModal.html

```diff
@@ -8,8 +8,12 @@
   </p>
 
   <div class="content">
-    <div class="fullEdit">
-      <textarea class="passageText" placeholder="Enter the body text of your passage here. To link to another passage, put two square brackets around its name, [[like this]]."></textarea>
+    <label for="edit-ds-oip">Mobile Commons Opt-in Path</label>
+    <input type="number" id="edit-ds-oip" name="ds-oip" class="block fillin" placeholder="Opt-in Path">
+
+    <div class="fullEdit" style="top: 8.5em;">
```

Comment:
In my PR, I added an `intertwine.css` css overrides file--helpful so we can avoid easily-forgotten inline styles.